docs/guides/basic-database.rst

```diff
@@ -67,7 +67,7 @@ Now, let's add a zone and some records::
     $ sudo -u pdns pdnsutil create-zone example.com ns1.example.com
     Creating empty zone 'example.com'
     Also adding one NS record
-    $ sudo -u pdns pdnsutil add-record example.com '' MX '25 mail.example.com'
+    $ sudo -u pdns pdnsutil add-record example.com @ MX '25 mail.example.com'
```

Comment:
the previous syntax here still works, right?

Reply:
Yes, `''` emits a warning however. `@` has the same effect but doesn't emit a warning.